src/ThreeEditor/js/Sidebar.Material.ZoneMaterial.js

```diff
@@ -125,12 +116,8 @@ export function SidebarZoneMaterial(editor) {
 	signals.objectSelected.add(object => {
 		let hasMaterial = false;
 
-		if (object && object.material) {
+		if (object && object.simulationMaterial) {
 			hasMaterial = true;
-
-			if (Array.isArray(object.material) && object.material.length === 0) {
-				hasMaterial = false;
-			}
 		}
 
 		if (hasMaterial) {
```

Comment:
Are you using `hasMaterial` somewhere else? Because if you need condition from 119 line just to determine this condition then why won't you use the same condition here and delete the one above?
```suggestion
		if (object?.simulationMaterial) {
```